docs/future-work/oss-communication-improvements/build-a-tutorial.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Build a Tutorial
+rfc: not-required
+estimated-scope: medium
+improved-metric: community-engagement
+skills: video-production, writing, python
+output-type: tutorial
+status: evergreen
```

Comment:
It still feels strange to have partial overlap between our general ways to contribute page and this future work roadmap. There are other larger items that are not on here, like writing blog posts, or building Monty demos, etc. and it feels a bit arbitrary now where we draw the line. 

We should also link to our existing page on [contributing tutorials](https://thousandbrainsproject.readme.io/docs/contributing-tutorials)